### ISSUE TITLE
Staging Review finding: Date selection is accessible during loading on iOS/VoiceOver

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111495]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

While the page is loading, users on iOS/VoiceOver can swipe or tap and access the calendar before the  loading state has finished.

Allowing users to interact with elements that should be disabled or unavailable can cause confusion, errors, and create accessibility barriers.

<details><summary>Link, screenshot or steps to recreate</summary>

(to be added date-loading-selection)

</details>

## Recommended action

While the page is loading and the calendar is dimmed, you should not be able to navigate through it. Alternatively, hide the calendar until loading is fully complete.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@rsmithadhoc can you can you confirm which URL this issue showed up on? We have this calendar in a couple of other flows too and want to check them

**Comment by outerpress** [comment]
@a6marinos @StevenStraily This is a duplicate of ADE https://github.com/department-of-veterans-affairs/va.gov-team/issues/111641?issue=department-of-veterans-affairs%7Cva.gov-team%7C111876
